Adds constructor for ComplateViewResolver with Nashorn as default scripting engine

plafue opened the pull request.

mvitz commented on src/main/java/com/github/complate/ComplateViewResolver.java:
```diff
@@ -20,15 +20,19 @@
     }
 
+    public ComplateViewResolver(final Resource bundle) {
+        Assert.notNull(bundle, "bundle must not be null");
```
I personally dislike more than one constructor which does real work if possible. Therefore I would implement this one as:
`this(new NashornScriptingBridge(), bundle);`

Because we are at such a low version we could also think about reversing the order of the two parameters because that would fit the normal java overloading conventions.

dwestheide replied:
I agree with @mvitz about always delegating to a primary constructor using `this`. I am also fine with changing the order of the parameters. Right now, we are still free to do these kinds of breaking changes.

plafue replied:
Alright! Thanks for the feedback. I'm on it

plafue committed:
Ensure that the global object is available within JavaScript

This change is effectively a polyfill (well, prollyfill anyway) - see
http://2ality.com/2016/09/global.html for details
